doc/lua_api.md

```diff
@@ -10409,10 +10409,9 @@ See [Decoration types]. Used by `minetest.register_decoration`.
 
     flags = "liquid_surface, force_placement, all_floors, all_ceilings",
     -- Flags for all decoration types.
-    -- "liquid_surface": Instead of placement on the highest solid surface
-    --   in a mapchunk column, placement is on the highest liquid surface.
-    --   Placement is disabled if solid nodes are found above the liquid
-    --   surface.
+    -- "liquid_surface": Find the highest liquid (not solid) surface under
```

Comment:
Is it liquid or it can also be airlike? Since your parenthesis

Reply:
As far as I can tell `liquid_type != LIQUID_NONE`